controller/workitem.go

```diff
@@ -582,6 +582,10 @@ func ConvertWorkItemsToCSV(ctx context.Context, app application.Application, wit
 	// the columns with a stable order
 	columnKeys := []string{}
 	columnLabels := []string{}
+	// add the WIT name manually
+	const witNameKey = "_type"
+	columnKeys = append(columnKeys, witNameKey)
+	columnLabels = append(columnLabels, "_Type")
```

Comment:
Why not add this column after the labels have been sorted? Then we can have it always in the first column which is a lot easier for manual or automatic processing.

Reply:
Fixed in  ea4d094